src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,83 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+
+  h1 {
+    color: $c-text-primary-white;
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+}
+
+.recommender-grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender__grid__material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  .button-favourite {
+    cursor: pointer;
```

Comment:
Please consider whether it is necessary to specify `cursor: pointer` here.

1. If we want the `.button-favourite` element to use the cursor pointer when I would expect us to want this everywhere. Not only within the Recommender. Thus this should go in [`button-favourite.scss`](https://github.com/reload/dpl-design-system/blob/main/src/stories/Library/Buttons/button-favourite/button-favourite.scss#L1-L1) instead.
2. Based on my testing in [the favourite button already has a pointer cursor](https://danskernesdigitalebibliotek.github.io/dpl-design-system/?path=/story/components-button-favourite--favourite&args=fill:false) as this is specified in [`icon-favourite`](https://github.com/reload/dpl-design-system/blob/main/src/stories/Library/Icons/icon-favourite/icon-favourite.scss#L1-L1).

Reply:
I'll fix